### PR TITLE
chore: update Links to Design Pattern > Assessment

### LINKED ISSUE #6116: Links to Design Pattern in Assessment take you to main ARIA APG page
## Describe the bug
The design pattern links no longer take you to the design pattern page

**To Reproduce**
Got to Assessment > Custom widgets > Design pattern on a page with custom widgets, note that the links we have in the list of instances no longer take you to the design pattern page. Instead they take you to the landing page with all design patterns

## Context (please complete the following information)

- OS Name & Version:  Windows 
- AI-Web Version & Environment:  Insider MV3  2.35.0
- Browser Version:  Chrome
- Target Page:  any

## Did you search for similar existing issues?
yes

## Additional context

There is a new site for WAI ARIA practices that is redirecting the old link to this.

### PR DESCRIPTION
**Pull request by Pranav-yadav**
- [x] point to correct ARIA APG links
- [x] fix various broken links w/ (typos)
- [x] fix unit tests for the same :)

#### Details
https://github.com/microsoft/accessibility-insights-web/issues/6116#issuecomment-1293832001
> This is the [landing page link](https://www.w3.org/WAI/ARIA/apg/patterns), we would need to point to the right subpage for each pattern. For example for combobox the current link is: https://www.w3.org/TR/wai-aria-practices-1.1/#combobox the new link should be: https://www.w3.org/WAI/ARIA/apg/patterns/combobox/

##### Motivation

closes #6116 

#### Pull request checklist

- [x] Addresses an existing issue: #6116 
- [-] Ran `yarn null:autoadd`
- [-] Ran `yarn fastpass`
- [x] Added/updated relevant unit test(s) (and ran `yarn test`)
- [x] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [-] (UI changes only) Added screenshots/GIFs to description above
- [-] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

**Review by waabid — APPROVED** [review]
LGTM 😄

File: src/assessments/custom-widgets/custom-widgets-column-renderer.tsx [hunk 1]
```diff
@@ -55,94 +55,90 @@ function makeFlatDesignPatternString(patterns: DesignPattern[]): string {
 }
 
 const roleToDesignPatternsMapping: DictionaryStringTo<DesignPattern[]> = {
-    alert: [{ designPattern: 'Alert', URL: 'https://www.w3.org/TR/wai-aria-practices-1.1/#alert' }],
+    alert: [{ designPattern: 'Alert', URL: 'https://www.w3.org/WAI/ARIA/apg/patterns/alert' }],
     alertdialog: [
         {
-            designPattern: 'Alert or Message Dialog',
-            URL: 'https://www.w3.org/TR/wai-aria-practices-1.1/#alertdialog',
+            designPattern: 'Alert and Message Dialogs',
```
Comment (ferBonnin) [inline src/assessments/custom-widgets/custom-widgets-column-renderer.tsx:61]: This should be Alert or Message (in this case the design pattern can be one or the other but not both

**Review by ferBonnin — CHANGES_REQUESTED** [review]
one minor change requested, otherwise looks good!

File: src/assessments/custom-widgets/custom-widgets-column-renderer.tsx (hunk 1 above)
Comment (Pranav-yadav) [reply src/assessments/custom-widgets/custom-widgets-column-renderer.tsx:61]: Actually, copied the title from w3.
Updating..

**Review by ferBonnin — APPROVED** [review]

**Comment by Pranav-yadav** [comment]
Looks like tests also needs update.
Updating..

**Comment by Pranav-yadav** [comment]
Andd this one isn't related to changes in this PR I guess? :-
[actions/runs/3347486791/jobs/5545523927#step:11:932](https://github.com/microsoft/accessibility-insights-web/actions/runs/3347486791/jobs/5545523927#step:11:932)

**Comment by Pranav-yadav** [comment]
Fixed the tests :)

PS: Lmk, how to resolve 👇 


> @ferBonnin Apart from this PR, I wasn't able to setup the local dev env for this repo, getting errors due to node version. PFA, (tried upgrading node, re-installing, cleaning caches, etc. still it didn't work) any guess? image

**Comment by ferBonnin** [comment]
> Andd this one isn't related to changes in this PR I guess? :- [actions/runs/3347486791/jobs/5545523927#step:11:932](https://github.com/microsoft/accessibility-insights-web/actions/runs/3347486791/jobs/5545523927#step:11:932)
> 
I re-ran that job and it looks ok now 🙂

**Comment by ferBonnin** [comment]
> @ferBonnin Apart from this PR, I wasn't able to setup the local dev env for this repo, getting errors due to node version. PFA, (tried upgrading node, re-installing, cleaning caches, etc. still it didn't work) any guess? image

tagging @lisli1 to see if we can help with this!

**Comment by lisli1** [comment]
> @ferBonnin Apart from this PR, I wasn't able to setup the local dev env for this repo, getting errors due to node version. PFA, (tried upgrading node, re-installing, cleaning caches, etc. still it didn't work) any guess? image

@Pranav-yadav we only support the version that we actively test against, which is currently 16. Please try using node version ^16.14.2 and see if that solves your issue. For managing different node installs side-by-side, you may try using Volta (on Windows) or nvm (on mac/linux/WSL).

**Comment by Pranav-yadav** [comment]
Thanks @lisli1 , got it.

**Comment by Pranav-yadav** [comment]
Hey @lisli1!
Are any further changes required? :)

**Pull request merged**